vm/jump_table_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2024 The VeChainThor developers
+
+// Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
+// file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
+
+package vm
+
+import (
+	"testing"
+
+	"github.com/holiman/uint256"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPush0OpCode(t *testing.T) {
```

Comment:
can we add similar test for BASEFEE ?

Reply:
BASEFEE is tested in runtime package, this file was aligned with upstream.